### PR TITLE
minor ui + utility bugs fixes

### LINKED ISSUE #236: [BUG] Minor UI + Utility bugs
### Tasks
- [ ] Add skeleton-loader in `Our Store`
- [ ] Update the Cart Model with real-data.

image
- [ ] We don't need the **Table header** and **Apply Coupon section** when cart is empty.

image
- [ ] Remove this `Contact Us` from here and directly route it through the `Contact` in `Navbar`

image

### PR DESCRIPTION
**Pull request by SandipGyawali**
- skeleton-loader added	in Our Store
- cart model updated with real-data with image,	price and name
- table	header and cupon removal for empty cart
- footer contact-us refactor used in the header-nav section

Fixes #236

### CODE REVIEW

**Review by pooranjoyb — COMMENTED** [review]
Great job, @SandipGyawali! 🎉 Thank you for submitting your pull request. Your contribution is valuable and we appreciate your efforts to improve our project.<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by pooranjoyb** [comment]
Deployment has failed, please check by running `npm run build` and debug accordingly @SandipGyawali

**Comment by SandipGyawali** [comment]
> Deployment has failed, please check by running `npm run build` and debug accordingly @SandipGyawali

@pooranjoyb i guess it's fixed

**Comment by Mahitej28** [comment]
@SandipGyawali your branch has merge conflicts in `Navbar.tsx` and Shop.tsx` kindly resolve, otherwise it can't be merged

File: src/components/Skeleton/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+function SkeletonWithContent() {
```
Comment (Mahitej28) [inline src/components/Skeleton/Content.tsx:1]: @pooranjoyb is there a need to create a separate Skeleton folder?, rest all looks good to me.

**Review by Mahitej28 — COMMENTED** [review]
@pooranjoyb have a look at it.

File: src/components/Skeleton/Content.tsx (hunk 1 above)
Comment (pooranjoyb) [reply src/components/Skeleton/Content.tsx:1]: Nope not needed, we can directly import and use skeletion wherever necessary. Please check @SandipGyawali

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

**Comment by pooranjoyb** [comment]
Also resolve the conflicts @SandipGyawali

**Review by Mahitej28 — APPROVED** [review]

**Review by pooranjoyb — APPROVED** [review]

**Pull request merged**